README.md

```diff
@@ -31,6 +31,12 @@ rust-openssl to a separate installation. OSX releases starting at 10.11, "El
 Capitan", no longer include OpenSSL headers which will prevent the `openssl`
 crate from compiling.
 
+Try the following commands to fix the issue (assumes Homebrew is used for package management):
+```
+brew install openssl
+brew link openssl --force
```

Comment:
It'd go in your `~/.bashrc` or equivalent for other shells. The `OPENSSL_LIB_DIR` and `OPENSSL_INCLUDE_DIR` variables are only used by this library's logic, while `brew link openssl --force` is a global action. The downside of that is that any programs that expect to be running against the OpenSSL 0.9.8 that OSX ships with could end up pulling in the modern one installed by homebrew and blowing up, or finding the *headers* for the homebrew version but linking against the stock version. 0.9.8 and 1.0.2 aren't binary compatible, so this will cause mysterious breakage.